### ISSUE TITLE
Compliance\Payment Notification - Create a payment notification

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-compliance#21]
#### Description:

As an external user, I can fill out a payment notification form so that I can make a monetary payment towards my compliance obligation 

[Figma](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20107-1077&t=jsaF4ZFPD8JLt1vE-1)
#### Acceptance Criteria:

- [ ] Given that I am on my compliance obligation summary page, when I click on the 'Create Payment Notification' tab, then I am brought to the Create a payment notification page

On the page is viewable:

- [ ] Create Payment Notification heading
- [ ] Instructions on how the payment notification works
- [ ] Display information (pulled from reporting/obligation summary/registration information): operation name, compliance period, Invoice # (Obligation ID), amount due (in dollars)
- [ ] Payment notification information (fillable fields): Estimated payment date (date picker), intended payment amount (field formatted for dollars)
- [ ] Button to add another payment notification (same fields as in the payment notification information), there is no cap on the number of notifications that can be created
- [ ] Back button to return to the Compliance Summary Screen
- [ ] Continue button to view the status of the payment.

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by #18 
- Blocking #26
